haec/prisma/schema.prisma

```diff
@@ -14,3 +14,20 @@ datasource db {
 }
 
 
+model User {
+  id    Int     @default(autoincrement()) @id
+  email String  @unique
+  name  String?
+
+  @@map("users")
+}
+
+model horses {
```

Comment:
By convention, Prisma models are uppercased and singular like `User` above. You'd use the `@@map()` directive to actually change the database table to be `snake_case` and plural. So your actual db query would look like:
```sql
select * from horses;
```
But the model in here would look like:
```
model Horses {
   ...
   
   @@map("horses")
}
```